apps/emqx_bridge_doris/src/emqx_bridge_doris_connector_schema.erl

```diff
@@ -0,0 +1,133 @@
+%%--------------------------------------------------------------------
+%% Copyright (c) 2025 EMQ Technologies Co., Ltd. All Rights Reserved.
+%%--------------------------------------------------------------------
+-module(emqx_bridge_doris_connector_schema).
+
+-feature(maybe_expr, enable).
+
+-behaviour(hocon_schema).
+-behaviour(emqx_connector_examples).
+
+-include_lib("typerefl/include/types.hrl").
+-include_lib("hocon/include/hoconsc.hrl").
+-include("emqx_bridge_doris.hrl").
+
+%% `hocon_schema' API
+-export([
+    namespace/0,
+    roots/0,
+    fields/1,
+    desc/1
+]).
+
+%% `emqx_connector_examples' API
+-export([
+    connector_examples/1
+]).
+
+%% API
+-export([]).
+
+%%------------------------------------------------------------------------------
+%% Type declarations
+%%------------------------------------------------------------------------------
+
+%%------------------------------------------------------------------------------
+%% `hocon_schema' API
+%%------------------------------------------------------------------------------
+
+namespace() ->
+    "connector_doris".
+
+roots() ->
+    [].
+
+fields(Field) when
+    Field == "get_connector";
+    Field == "put_connector";
+    Field == "post_connector"
+->
+    emqx_connector_schema:api_fields(Field, ?CONNECTOR_TYPE, fields(connector_config));
+fields("config_connector") ->
+    emqx_connector_schema:common_fields() ++ fields(connector_config);
+fields(connector_config) ->
+    Fields =
+        lists:map(
+            fun
+                ({server = K, Sc}) ->
+                    Override = #{
+                        %% to please dialyzer...
```

Comment:
:)

maybe (not this PR) add a new type spec for `hocon_schema:override/2` to allow missing `type`